ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/IndentationRule.kt

```diff
@@ -765,13 +766,24 @@ class IndentationRule : Rule("indent"), Rule.Modifier.RestrictToRootLast {
     }
 
     private fun adjustExpectedIndentAfterColon(n: ASTNode, ctx: IndentContext) {
-        expectedIndent++
-        debug { "++after(COLON) -> $expectedIndent" }
-        if (n.isPartOf(FUN)) {
-            val returnType = n.nextCodeSibling()
-            ctx.exitAdjBy(returnType!!, -1)
-        } else {
-            ctx.exitAdjBy(n.treeParent, -1)
+        when {
+            n.isPartOf(FUN) -> {
+                expectedIndent++
+                debug { "++after(COLON IN FUN) -> $expectedIndent" }
+                val returnType = n.nextCodeSibling()
+                ctx.exitAdjBy(returnType!!, -1)
+            }
+            n.treeParent.isPartOf(SECONDARY_CONSTRUCTOR) -> {
+                expectedIndent += 2
```

Comment:
```suggestion
                expectedIndent++
```
We don't support continuation indent at the moment.

The reason you've got this code indented as you mentioned in the original issue is that you're using `ij_kotlin_continuation_indent_in_supertype_lists` probably.